custom_components/google_home/api.py

```diff
@@ -227,3 +229,94 @@ async def update_google_devices_information(self) -> List[GoogleHomeDevice]:
             ]
         )
         return coordinator_data
+
+    async def delete_alarm_or_timer(
+        self, device: GoogleHomeDevice, item_to_delete: str
+    ) -> None:
+        """Deletes a timer or alarm.
+        Can also delete multiple if a list is provided (Not implemented yet)."""
+
+        if device.ip_address is None:
+            _LOGGER.error("Device %s doesn't have IP address!", device.name)
```

Comment:
I think this actually should be warning. Errors should be used for things which shouldn't happen, warnings should be used for things which may happen if used incorrectly, e.g. calling delete service for offline device.
```suggestion
            _LOGGER.warning("Device %s doesn't have IP address!", device.name)
```